crates/nargo_cli/src/manifest.rs

```diff
@@ -15,19 +15,21 @@ use crate::{errors::ManifestError, git::clone_git_repo};
 #[derive(Debug, Deserialize, Clone)]
 struct PackageConfig {
     package: PackageMetadata,
-    dependencies: BTreeMap<String, DependencyConfig>,
+    dependencies: Option<BTreeMap<String, DependencyConfig>>,
```

Comment:
```suggestion
    #[serde(default)]
    dependencies: BTreeMap<String, DependencyConfig>,
```

Thoughts on this solution instead? This keeps our own logic cleaner imo

Reply:
I generally like to be explicit when things are optional, but I don't have a preference here. I'll try it locally and push

Reply:
Done!